### PR TITLE
Juan dev little

### PR DESCRIPTION
**Pull request by jazcarretao**
Update the README.md and requirements.txt to agree with the new requirements. Are they correct or is it even necessary to include them in the README.md?
PD: I called the branch "piano" because in Italian means "little" because in this branch I just made little changes. I will choose a better word next time.

### CODE REVIEW

**Comment by ebezzam** [comment]
Thanks @jazcarretao! 

@fakufaku, I actually had a question regarding this because would you consider `sounddevice` to be a requirement of `pyroomacoustics` if it is only used in an example script? (not sure if that's also the case for  `mir_eval`.)

**Comment by fakufaku** [comment]
Thanks @jazcarretao 
Good point @ebezzam , in fact I'd prefer not to grow the requirements list. Sounddevice is used now only in the examples and also conditionaly in a few places (e.g. datasets.audiosample), by conditionaly, I mean that if it is not installed, no function will fail. Instead, it will display an error message that tells the user to install sounddevice to get the functionality.

Ideally, I'd like to get rid of joblib too (that is only used in FRIDA, and probably not really unavoidable).

mir_eval is only used in examples and should not be a requirement. Cython is now indeed needed.

How about indicating optional dependencies in the README, and leave them out of the requirements file ?

**Comment by jazcarretao** [comment]
I agree with that point.
I can update the README.md to explain which requirements need to be installed in case someone wants to run the examples.

File: requirements.txt [hunk 1]
```diff
@@ -4,3 +4,5 @@ scipy>=0.18.0
 matplotlib
 joblib
 requests
+sounddevice
```
[the remark this answers is not in the document]
Comment (fakufaku) [reply requirements.txt]: Correct

File: README.rst [hunk 2]
```diff
@@ -111,6 +111,13 @@ The requirements are::
 * numpy 
 * scipy 
 * matplotlib
+* Cython
+
+In additions, these packages are installed to run the example scripts::
```
Comment (fakufaku) [inline README.rst]: Very minor remark: "... these packages are **needed** to run ..."

**Comment by fakufaku** [comment]
I finally addressed all this together with reducing the number of package required in PR #55 .

**Pull request closed without merging**